AutomatedTesting/Gem/PythonTests/prefab/PrefabLevel_BasicWorkflow.py

```diff
@@ -7,17 +7,19 @@
 
 # fmt:off
 class Tests():
-    create_new_entity =       ("Entity: 'CreateNewEntity' passed",                          "Entity: 'CreateNewEntity' failed")
-    create_prefab =           ("Prefab: 'CreatePrefab' passed",                             "Prefab: 'CreatePrefab' failed")
-    instantiate_prefab =      ("Prefab: 'InstantiatePrefab' passed",                        "Prefab: 'InstantiatePrefab' failed")
-    new_prefab_position =     ("Prefab: new prefab's position is at the expected position", "Prefab: new prefab's position is *not* at the expected position")
+    create_new_entity =       ("Entity: 'CreateNewEntity' passed",                           "Entity: 'CreateNewEntity' failed")
+    create_prefab =           ("Prefab: 'CreatePrefab' passed",                              "Prefab: 'CreatePrefab' failed")
+    instantiate_prefab =      ("Prefab: 'InstantiatePrefab' passed",                         "Prefab: 'InstantiatePrefab' failed")
+    new_prefab_position =     ("Prefab: new prefab's position is at the expected position",  "Prefab: new prefab's position is *not* at the expected position")
+    delete_prefab =           ("Prefab: 'DeleteEntitiesAndAllDescendantsInInstance' passed", "Prefab: 'DeleteEntitiesAndAllDescendantsInInstance' failed")
 # fmt:on
 
 def PrefabLevel_BasicWorkflow():
     """
     This test will help verify if the following functions related to Prefab work as expected: 
     - CreatePrefab
     - InstantiatePrefab
+    - DeleteEntitiesAndAllDescendantsInInstance
```

Comment:
Instead of adding more steps to this test, could we create a new test?
That way we can have more granularity and have more information in the case of test failures

Reply:
We want to just cover very simple test cases here first to make sure all the needed prefab apis are exposed to python and work as expected. After that, we will add more complicated tests for verification.

Reply:
I agree with @AMZN-AlexOteiza here. Creating separate tests for different basic workflows like create, instantiate, delete, duplicate, detach, reparent would help with 2 things:
1. It won't result in one huge test file
2. It would allow us to test different use cases within the basic workflows like (create ->detach), (instantiate->delete), (create->instantiate->duplicate) etc.

Reply:
Will do that in the next PR.